core/ui/ControlPanel/Modals/AddPlugins.tid

```diff
@@ -5,7 +5,7 @@ subtitle: {{$:/core/images/download-button}} {{$:/language/ControlPanel/Plugins/
 
 \define install-plugin-button()
 <$button>
-<$action-sendmessage $message="tm-load-plugin-from-library" url={{!!url}} title={{$(assetInfo)$!!original-title}}/>
```

Comment:
No, the "title" parameter is needed to indicate which plugin should be loaded. I think the deletion in your pull request was a typo: the line below has a syntax error.